README.md

```diff
@@ -449,16 +457,25 @@ be requested if the current time is later than the cached token expiry time minu
 | lookup.partial-cache.cache-missing-key                        | optional | This is a boolean that defaults to true. Whether to store an empty value into the cache if the lookup key doesn't match any rows in the table. `lookup.cache` must be set to `PARTIAL` to use this option. See the following <a href="#lookup-cache">Lookup Cache</a> section for more details.                                                                   |
-| lookup.max-retries                                            | optional | The max retry times if the lookup failed; default is 3. See the following <a href="#lookup-cache">Lookup Cache</a> section for more details.                                                                                                                                                                                                                      |
-| gid.connector.http.lookup.error.code                          | optional | List of HTTP status codes that should be treated as errors by HTTP Source, separated with comma.                                                                                                                                                                                                                                                                  |
-| gid.connector.http.lookup.error.code.exclude                  | optional | List of HTTP status codes that should be excluded from the `gid.connector.http.lookup.error.code` list, separated with comma.                                                                                                                                                                                                                                     |
+| lookup.retry-strategy.type                                    | optional | Defines the retry strategy to use in case of lookup failures. Accepted values are: `none` (default), `fixed-delay` and `exponential-delay`.                                                                                                                                                                                                                       |
+| lookup.retry-strategy.fixed-delay.attempts                    | optional | The number of times that connector retries lookup execution before connector returns empty result.                                                                                                                                                                                                                                                                |
```

Comment:
> returns empty result

In current implementation, an empty result is returned if error status code is returned or when exception is thrown. To be honest I'm strongly against such approach. Errors should not be suppressed, at least by default. If error suppression is needed, user should have to configure it explicitly. WDYT?